org.linuxshowplayer.LinuxShowPlayer.json

```diff
@@ -0,0 +1,226 @@
+{
+    "app-id": "org.linuxshowplayer.LinuxShowPlayer",
```

Comment:
the domain `linuxshowplayer.com` doesn't seem to exist, therefor the app-id doesn't meet the requirements.

Reply:
It's `linuxshowplayer.org`